daemon/daemon.go

```diff
@@ -479,6 +479,7 @@ func (daemon *Daemon) newContainer(name string, config *runconfig.Config, imgID
 	var (
 		id  string
 		err error
+		ane = name == ""
```

Comment:
Did not want to reset indent of previous vars. Being a function's internal var thought it was ok also because it is getting assigned to `IsAnonymousEndpoint`, self explicative.

If you think it is still better to change it, any suggestion ? I can only think of `isAnonymous` or `nameNotSpecified`